<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>430</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Garamond</family>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Photoshop</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>0</y>
      <width>571</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Filter</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>201</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radio_average">
             <property name="text">
              <string>Average</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radio_median">
             <property name="text">
              <string>Median</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radio_gaussian">
             <property name="text">
              <string>Gaussian</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radio_laplace">
             <property name="text">
              <string>Laplace</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>61</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>kernel</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Btn_kernelRun">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>130</y>
            <width>61</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>120</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Channel</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>261</width>
            <height>151</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>B</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_b">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>G</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_g">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,7">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>R</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_r">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,7">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Brightness</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_brightness">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="3,7">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Contrast</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="slider_contrast">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="singleStep">
                <number>0</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="sliderPosition">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>cvtColor</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>251</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QRadioButton" name="radio_cvtGray">
             <property name="text">
              <string>Gray</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="radio_cvtHSV">
             <property name="text">
              <string>HSV</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radio_cvtHSL">
             <property name="text">
              <string>HSL</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="radio_cvtYCrCb">
             <property name="text">
              <string>YCrCb</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="Btn_cvt">
             <property name="text">
              <string>Change</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>File Perspective</string>
         </property>
         <widget class="QPushButton" name="Btn_findCorner">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>30</y>
            <width>101</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>FindCorner</string>
          </property>
         </widget>
         <widget class="QPushButton" name="Btn_persChange">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>100</y>
            <width>101</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionUndo"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
